package.json

```diff
@@ -48,6 +48,11 @@
     "exclude": [
       "dist/**/*.spec.js",
       "dist/**/*.d.ts"
+    ],
+    "all": true,
```

Comment:
https://www.npmjs.com/package/nyc#include-reports-for-files-that-are-not-required